lib/android/src/main/java/com/reactnativeldk/LdkModule.kt

```diff
@@ -789,37 +805,11 @@ class LdkModule(reactContext: ReactApplicationContext) : ReactContextBaseJavaMod
             return handleResolve(promise, LdkCallbackResponses.invoice_payment_success)
         }
 
-        val error = res as? Result_ThirtyTwoBytesPaymentErrorZ.Result_ThirtyTwoBytesPaymentErrorZ_Err
-
-        val invoiceError = error?.err as? PaymentError.Invoice
-        if (invoiceError != null) {
-            return handleReject(promise, LdkErrors.invoice_payment_fail_invoice, Error(invoiceError.invoice))
-        }
-
-        val sendingError = error?.err as? PaymentError.Sending
-        if (sendingError != null) {
-            val paymentAllFailedResendSafe = sendingError.sending as? PaymentSendFailure.AllFailedResendSafe
-            if (paymentAllFailedResendSafe != null) {
-                return handleReject(promise, LdkErrors.invoice_payment_fail_resend_safe, Error(paymentAllFailedResendSafe.all_failed_resend_safe.map { it.toString() }.toString()))
-            }
-
-            val paymentParameterError = sendingError.sending as? PaymentSendFailure.ParameterError
-            if (paymentParameterError != null) {
-                return handleReject(promise, LdkErrors.invoice_payment_fail_parameter_error, Error(paymentParameterError.parameter_error.toString()))
-            }
-
-            val paymentPartialFailure = sendingError.sending as? PaymentSendFailure.PartialFailure
-            if (paymentPartialFailure != null) {
-                return handleReject(promise, LdkErrors.invoice_payment_fail_partial, Error(paymentPartialFailure.toString()))
-            }
-
-            val paymentPathParameterError = sendingError.sending as? PaymentSendFailure.PathParameterError
-            if (paymentPathParameterError != null) {
-                return handleReject(promise, LdkErrors.invoice_payment_fail_path_parameter_error, Error(paymentPartialFailure.toString()))
-            }
+        val error = res as? Result_NoneRetryableSendFailureZ_Err
+            ?: return handleReject(promise, LdkErrors.invoice_payment_fail_unknown)
 
-            return handleReject(promise, LdkErrors.invoice_payment_fail_sending, Error("PaymentError.Sending"))
-        }
+//        val sendingError = error.err as? PaymentSendFailure.DuplicatePayment
+        //TODO try cast to get error types
```

Comment:
@ovitrif

Reply:
I believe it's one of these 3 enum entries according to Android Studio
```kt
RetryableSendFailure.LDKRetryableSendFailure_DuplicatePayment
RetryableSendFailure.LDKRetryableSendFailure_PaymentExpired
RetryableSendFailure.LDKRetryableSendFailure_RouteNotFound
```